hashing/te-tag-query-python/all_in_one_lib/all_in_one.py

```diff
@@ -0,0 +1,132 @@
+##!/usr/bin/env python
+
+"""
+A wrapper around multi-stage ThreatExchange operations.
+
+Includes simple matching and writing back. Useful for quickly validating new
+sources of ThreatExchange data. A possible template for a native
+implementation in your own architecture.
+
+This helper heavily relies on a config file to provide consistent behavior
+between stages, and a state file to store hashes.
+"""
+
+import argparse
+import os
+import os.path
+import pathlib
+import sys
+import typing as t
+
+import TE
+
+from .collab_config import CollaborationConfig
+from .dataset import Dataset
+from .commands import base, label, match, fetch
+
+
+def get_subcommands() -> t.List[t.Type[base.Command]]:
```

Comment:
A note on typing: I have no idea how to get typing to run in an open source way. I tried both mypy and pyre on my devserver and it tried to run them in an FB context, and then did nothing because it wasn't configured correctly. There's bound to be tons of typing errors that will only be discovered once we can get those to be run.

Reply:
Okay, was able to get help from @bodnarbm to run locally. There are so many issues in the first diff that I'm going to juts punt it to a followup I think.